src/bindings/js/node/include/core_set_property_args.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <napi.h>
+
+#include "node/include/helper.hpp"
+#include "openvino/runtime/core.hpp"
+
+/**
+ * @brief This struct retrieves data from Napi::CallbackInfo.
+ */
+struct CoreSetPropertyArgs {
```

Comment:
Maybe I don't see purpose for this struct but from usage it looks like could be replaced by set of functions like
- validate
- get_paramters
- get_device_name

or maybe function which tries get some property from `CallbackInfo` or throw.
 - try_get_paramters
 - try_get_device_name

Reply:
Main purpose was to improve readability and do not full core_wrap.cpp by extra helpers. But I agree, structure in separate file looks redundant. 
Refactored.